packages/components/src/components/header/header.styles.ts

```diff
@@ -52,7 +52,7 @@ export default css`
    */
   .header__logo syn-icon,
   .header__logo syn-icon::part(svg) {
-    color: var(--syn-color-primary-600);
+    color: var(--syn-logo-color);
```

Comment:
we need the fallback token here


```suggestion
    color: var(--syn-logo-color, var(--syn-color-primary-600));
```